async/as/Pin.h

```diff
@@ -0,0 +1,29 @@
+#ifndef ASYNC_AS_PIN_H
+#define ASYNC_AS_PIN_H
+
+#include <pthread.h>
+
+namespace async::as {
+struct CpuMask {
+#ifndef __APPLE__
+  cpu_set_t set;
+#endif // __APPLE__
+
+  inline void getaffinity_np([[maybe_unused]] pthread_t thread) {
```

Comment:
Why is this one not const, but the other one is?

Reply:
Because getaffinity overwrites the set?